lib/release_notes_bot_web/controllers/webhook_controller.ex

```diff
@@ -87,10 +87,12 @@ defmodule ReleaseNotesBotWeb.WebhookController do
     # Persist to persistence provider
     if action in @persist_actions do
       if match_url == @source_adv_repo_url do
-        Persists.persist(release["name"], release["body"], @source_adv_confluence)
+        Persists.persist(build_persistence_title(release), release["body"], @source_adv_confluence)
       else
-        Persists.persist(release["name"], release["body"])
+        Persists.persist(build_persistence_title(release), release["body"])
       end
     end
   end
+
+  defp build_persistence_title(release), do: release["name"]
```

Comment:
what is the purpose of offloading this lookup into this private function? is it because we persist the value as a title and we want a function that helps explain that name maps to title?

Reply:
Yes, the title was a touchy subject. Before pushing this PR, we went through many naming schemas. Having this function kept everything consistent. If we need to change the title, it can be done in one place - this function. `build_persistence_title` just returns a string that can be used for the title of the Persistence provider's release.